examples/gocounter/kubernetes-deployment/gocounter.yaml

```diff
@@ -1,87 +1,48 @@
----
-apiVersion: v1
-kind: ConfigMap
+apiVersion: apps/v1
```

Comment:
Discussed in meeting , just condense this + all other kube stuff into a single yaml